src/main/java/seedu/address/model/contact/Telegram.java

```diff
@@ -25,6 +28,8 @@
      * @param telegramUsername Telegram username.
      */
     public Telegram(String telegramUsername) {
```

Comment:
I like the defensive programming error checks here.

Reply:
Thank you 👍